### DISCUSSION TITLE
Can I Select all and Omit just few fields or just a field in a query?

### DISCUSSION THREAD

**Discussion by Mrbeyond** [go-gorm/gorm#6010] [Q&A]
Model:

type Test struct{
  Name string `json:"name"`
  Score  uint       `json:"score"`
  Subject  string       `json:"subject"`
  School  string       `json:"school"`
  Remark  string       `json:"remark"`
}

Can I achieve some thing like:

  var tests []Test
  
   db.select("tests.*").
	Omit("remark").
	Find(&tests)
	
Or :

    db.Omit("remark").
      Find(&tests)
      
      
 I don't want to select each field explicitly because I don't want a particular field to fetched on a particular endpoint

**Comment by gjlmotea** [comment]
same here
